Automattic-Tracks-iOS.podspec

```diff
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'Automattic-Tracks-iOS'
-  s.version       = '0.12.1-beta.2'
+  s.version       = '0.13.0-beta.1'
```

Comment:
The addition of the new, optional, configuration value for performance monitoring grants a minor version bump.